sunpy/timeseries/timeseriesbase.py

```diff
@@ -140,23 +138,27 @@ def __init__(self, data, meta=None, units=None, **kwargs):
 
     @property
     def source(self):
-        """Returns a string/object used to specify the source class of the TimeSeries."""
+        """
+        A string/object used to specify the source class of the TimeSeries.
```

Comment:
Properties show up as attributes in the docs, so don't need a "Returns..." at the beginning of the description.